srcpkgs/m4/template

```diff
@@ -14,6 +14,10 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
 checksum=f2c1e86ca0a404ff281631bdc8377638992744b175afb806e25871a24a934e07
 
+pre_build() {
+    sed -i '1 i @documentencoding ISO-8859-1' doc/m4.texi
+}
+
```

Comment:
And this one is ok for the chroot case.

Reply:
and ditto here, if the host contains texinfo, the build will fail.

Reply:
Ok. Will you push the fix or shall I?

Reply:
go ahead

Reply:
A problem with this is, if I modify the `doc/m4.texi` file, the build system insists on requiring `makeinfo` to rebuild the info files :) I could try to keep the previous modification timestamp for that file to fool the make rules...

Reply:
... which doesn't help because in that case the documentencoding is not propagated. So I think we cannot have this in the bootstrap, but only when `$CHROOT_READY` and with the host dependency on `texinfo`.

Reply:
another way would be forcing configure to stop detecting it...

Reply:
`MAKEINFO= ./configure ...` should do it?

Reply:
I don't think so. The change in `doc/m4.texi` actually needs to be seen by `makeinfo` to have any effect. Just patching the file and not using / translating it makes no sense AFAICT.

Reply:
What is `m4` even required for in the bootstrap? 

@xtraeme I was not alerted that I didn't have makeinfo until the `make` of m4 got to the step using makeinfo - I believe the configure script doesn't care about texinfo, and doesn't modify the build step to not use makeinfo.

My opinion would be just removing the docs from the bootstrap entirely is the best solution, and rebuilding it in the chroot.

Reply:
@fosslinux I also think so. We should put up a list of packages which need to be rebuilt after a bootstrap, probably in the Manual.md section about bootstrapping.

Reply:
I agree.